packages/x-license/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license",
-  "version": "7.0.0-beta.2",
+  "version": "7.0.0-beta.6",
```

Comment:
I would have voted for consistency with previous approaches before a consensus is reached, but if you decided otherwise, so be it. 👌